docs/donejs.mustache

```diff
@@ -52,7 +52,7 @@
                     <h2><span class="light">Improved App</span><br>Usability</h2>
                     <p>
                       <a href="./Features.html#ios-android-and-desktop-builds">iOS, Android, and Desktop Builds</a>;
-                      <a href="./Features.html#supports-all-browsers-even-ie8">Supports All Browsers, even IE8</a>;
+                      <a href="./Features.html#supports-all-browsers-even-ie9+">Supports All Browsers, even IE9+</a>;
```

Comment:
Should this include the `+` or not, i.e. `supports-all-browsers-even-ie9` vs. `supports-all-browsers-even-ie9+`?